subprojects/parseq/src/test/java/com/linkedin/parseq/ParSeqUnitTestHelperTest.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.parseq;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertThrows;
+
+public class ParSeqUnitTestHelperTest {
```

Comment:
Can we have a test for `ParSeqUnitTestHelper` using Junit5?

Reply:
Problem is that it won't be picked up by the build. For the entire "parseq" module we should use one testing framework.

It's fine to continue using TestNG for parseq project. However, we should make sure that parseq customers can use JUnit (hence this PR :-D)